code/game/machinery/washing_machine.dm

```diff
@@ -65,6 +65,13 @@
 			for(var/obj/item/clothing/I in contents)
 				I.color = wash_color //Simply recolor the items.
 
+				var/colors = list("light brown", "brown", "yellowgreen", "darkred", "lightred", "maroon", "red", "orange", "rainbow", "lightgreen", "green", "lightpurple", "purple", "gold", "darkblue", "lightblue", "aqua", "blue", "yellow", "black", "grey", "gray", "white", "latex", "nitrile", "budget insulated", "insulated", "captain's")
+				
+				for(var/old_color in colors)
+					if(old_color != wash_color)
```

Comment:
I'm looping through a list of common colours used for various clothes like jumpsuits, hats, gloves, shoes, bandanas, etc.

Then I just replace that word with the wash color. On the one hand it an item's name contains more than one colour in it's name they'll both get changed to wash_color, but on the other hand I don't see very many of those items.

I guess the if statement is redundant, but it's basically excluding the potential one case where the old_color matches the wash_color